indra/newview/llspatialpartition.cpp

```diff
@@ -1715,13 +1708,11 @@ void renderOctree(LLSpatialGroup* group)
 			for (LLSpatialGroup::element_iter i = group->getDataBegin(); i != group->getDataEnd(); ++i)
 			{
 				LLDrawable* drawable = (LLDrawable*)(*i)->getDrawable();
-				if(!drawable || drawable->getNumFaces() == 0)
+				if (!drawable || !drawable->getNumFaces() || !check_rigged_group(drawable))
 				{
```

Comment:
don't call check_rigged_group outside of llassert -- it's very slow, so we don't want to call it outside of dev builds